README.md

```diff
@@ -1 +1,175 @@
-# hello-rails-back-end
\ No newline at end of file
+<h1>Hello Rails React FRONT END  </h1>
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Rails React] <a name="about-project"></a>
+
+**[Hello Rails React]**
+
+This application utilizes Ruby on Rails as the backend and React as the frontend. The exchange of information between the two sides is facilitated through an API. 
+[This is the link of the backend application](https://github.com/Nessrine88/hello-react-front-end)
```

Comment:
- [ ] I think there is a typo error here, this link actually redirect to the front end and not backend.
```suggestion
[This is the link of the front end application](https://github.com/Nessrine88/hello-react-front-end)
```